force-app/main/default/chat/lwc/messageHistory/messageHistory.js

```diff
@@ -0,0 +1,28 @@
+import {api, LightningElement} from 'lwc';
+
+export default class MessageHistory extends LightningElement {
+    /**
+     *
+     * @type MessageWrapper[]
+     */
+    @api _messages;
+
+    @api
+    set messages(messages) {
+        if (!messages) {
+            throw new Error();
+        }
+
+        this._messages = messages;
+    }
+
+    get messages() {
+        return this._messages;
+    }
+
+    renderedCallback() {
+        const scrollArea = this.template.querySelector('.slds-scrollable_y');
+        scrollArea.scrollTop = scrollArea.scrollHeight;
```

Comment:
Could you please add a comment regarding what this line does?

Reply:
added a new method 8d870610e229cc8a420aa5daeddcdb7010ca08a5